src/components/Copyable.jsx

```diff
@@ -5,6 +5,7 @@ import React, { useState, useEffect } from "react";
 import CopyToClipboard from "react-copy-to-clipboard";
 import { FileCopy } from "@material-ui/icons";
 import { IconButton, makeStyles } from "@material-ui/core";
+import { useSnackbar } from "notistack";
```

Comment:
Wow, cool. Hooks for the snackbar!

Reply:
Or at least that's what it looks like it's doing...

Reply:
Yep!